packages/react/src/themes/teams/components/Dropdown/dropdownStyles.ts

```diff
@@ -32,19 +67,14 @@ const dropdownStyles: ComponentSlotStylesInput<DropdownProps & DropdownState, Dr
   }),
 
   triggerButton: ({ props: p, variables: v }): ICSSInJSStyle => {
-    const transparentColorStyle = {
-      backgroundColor: 'transparent',
-      borderColor: 'transparent',
-    }
     return {
       boxShadow: 'none',
       margin: '0',
       justifyContent: 'left',
       padding: v.comboboxPaddingButton,
       height: pxToRem(30),
-      ...transparentColorStyle,
       ...(p.multiple && { minWidth: 0, flex: 1 }),
-      ':hover': transparentColorStyle,
+      ...transparentColorStyleObj,
       ':focus': {
         ...transparentColorStyle,
```

Comment:
Do we really need to apply here again the `transparentColorStyle`?

Reply:
it's either that or writing a deep merge function...
for `:focus` pseudoselector there are even more styles that we need to override;

everything would be easier if we had a basic `Button`. Not sure what else I can do here at the moment

Reply:
I understand the issue, thanks.